src/component/feature/FeatureStrategy/FeatureStrategyIcon/FeatureStrategyIcon.tsx

```diff
@@ -18,7 +18,7 @@ export const FeatureStrategyIcon = ({
     return (
         <StyledIcon>
             <Tooltip title={formatStrategyName(strategyName)} arrow>
-                <div id={id} role="tooltip">
+                <div id={id}>
```

Comment:
What's happening here? Are you saying that the Icon is labelled by a div that's otherwise empty? Or is there some code in the `Icon` component that describes the icon's purpose?